README.md

```diff
@@ -68,7 +71,7 @@ VM_EXISTS_CODE=$?
 set -e
 
 STEP="Checking if machine $VM exists"
-if [ $VM_EXISTS_CODE -eq 1 ]; then
+if [ $VM_EXISTS_CODE -ne 1 ]; then
```

Comment:
Can you explain how this works? because It seems a bit counter intuitive.

Reply:
This judgement is wrong

Reply:
@VAllens What do you mean?

Reply:
@pecigonzalo 
`-if [ $VM_EXISTS_CODE -eq 1 ]; then`
There is no problem with this snippet.
"-eq" is "equal" mean, compared to "=", it is more biased towards numerical comparisons.

Reply:
Because of the syntax error of this `grep \""${VM}"\"`, the value of the parameter `VM_EXISTS_CODE` that causes the code snippet `VM_EXISTS_CODE=$?` is always `1`.
Actually, what we want is...
When the `default` instance does not exist, the `VM_EXISTS_CODE` parameter value is `1`, because the `grep` command executes regular expressions, it no have matches valid value and throw the exception.
When the `default` instance exists, after the execution of the `grep` command, `$?` Returns `0`.
## So we want to fix this syntax error and let it return `0` when the `default` instance has exists.

**PS** : `$?` means to show the exit state of the last command. `0` indicates no error, and the other indicates an error.